app.js

```diff
@@ -0,0 +1,64 @@
+const express = require("express");
+const cors = require("cors");
+const app = express();
+const bodyParser = require("body-parser");
+const port = 3000;
+const School = require("./School");
+const Student = require("./Student");
+
+app.use(cors());
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
+
+let mySchool = new School();
+
+// Create a Class w/ Course Name and Teacher
+app.post("/class", (req, res) => {
+    try {
+        let className = req.body.name;
+        let classTeacher = req.body.teacher;
+        mySchool.addClass(className, classTeacher);
+        res.status(200).json({
+            class: {className,classTeacher,students: []},
+            status: "success",
+            message: "new class added"
+        });
+    } catch (err) {
+        console.log(err)
```

Comment:
If the attempt to add class did not go well, you should send a `res.json` with a specific error message , so you could grab it on the frontend and display it to the user